cron/worker/main.go

```diff
@@ -51,6 +46,7 @@ func processRequest(ctx context.Context,
 	// This will reduce usage of the API.
 	delete(checksToRun, "Branch-Protection")
 
+	repoURLs := make([]repos.RepoURL, 0, len(batchRequest.GetRepos()))
```

Comment:
nit: initialize closer to the code using it. (before line 62)

Reply:
repoURLs is populated in the below for loop - line 58.